activiti-engine/src/main/java/org/activiti/engine/repository/DiagramLayout.java

```diff
@@ -1,74 +0,0 @@
-/* Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
@Salaboy 

Is it better to return this class to the engine? This class returns the coordinates on the image of each element from the images of XML and png.It is necessary to customize the generated png image.

However, it will be SVG instead of png and I think that it is not necessary much.So I removed it from engine.